spec/helpers/moods_helper_spec.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+describe MoodsHelper do
+  let(:user) { create(:user) }
+  let(:mood) { create(:mood, user_id: user.id) }
+  let(:no_form_tag) { {noFormTag: true} }
+  let(:mood_props) do
+    {
+      inputs: [
+        {
+          id: 'mood_name',
+          type: 'text',
+          name: 'mood[name]',
+          label: t('common.name'),
+          value: @mood&.name || nil,
+          required: true,
+          dark: true
+        },
+        {
+          id: 'mood_description',
+          type: 'textarea',
+          name: 'mood[description]',
+          label: t('common.form.description'),
+          value: @mood&.description || nil,
```

Comment:
If we are using the mood we just created, isn't it safe to assume it will always have a `description` and remove the safe navigation operator?